test/ResearcherContract.test.js

```diff
@@ -127,4 +140,36 @@ contract("ResearcherContract", (accounts) => {
       assert.equal(researcherExists, false);
     });
   });
+
+  context("when add a research", () => {
+    it("should return error when is not a researcher", async () => {
```

Comment:
Vc pode colocar os its dentro de um contexto que faça sentido para eles. Assim vc deixa seu código mais claro e fácil de manutenção.

Aqui vc dois contexto principais

`When is a researcher` e `When is not a researcher`

Reply:
corrigido